style.css

```diff
@@ -444,6 +444,20 @@ fxos-tv-button[type="circle-text"].danger.pressed {
   transition-property: transform, color, background-color;
 }
 
+fxos-tv-button[type="circle-text"].disabled.focused {
+  transform: scale(1.2);
+  color: white;
+  background-color: #5f6368;
+  transition-property: transform, color, background-color;
+}
+
+fxos-tv-button[type="circle-text"].disabled.pressed {
+  transform: scale(1.2);
```

Comment:
Thanks for the reminder! It's different from the previous one, this line is necessary to prevent the disabled style from being pressed done.